<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qalculator</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius:20px; </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft New Tai Lue</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>1666666</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 200);
border-radius: 25px;
border: 0; 
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,165, 0, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="opMulti">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,165, 0, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,165, 0, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,165, 0, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalBtn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,165, 0, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backBtn">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>130</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 200);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decimalBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num0">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>370</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="num4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(91, 91, 91, 100);
border-radius: 25px;
border: 0; </string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>289</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
